src/ragas/executor.py

```diff
@@ -96,7 +99,6 @@ def run(self):
             results = self.loop.run_until_complete(self._aresults())
         finally:
             self.results = results
-            self.loop.stop()
```

Comment:
why did you remove this?

Reply:
Thanks a lot for putting the effort to fix this issue ❤️ 🙂

Reply:
Because if you use get_event_loop from a jupyter notebook you obtain the loop of the own jupyter, thus closing it might raise some problems. There might be another solution but I thought it was the most straightforward one.

Reply:
Thanks to you for creating such an awesome library!

Reply:
> Because if you use get_event_loop from a jupyter notebook you obtain the loop of the own jupyter, thus closing it might raise some problems.

Thank you so much for this! 🙇‍♂️

Reply:
Hi @abetatos .  Thanks for submitting the PR.  🙏  .   I tried to follow this issue but my knowledge of `asyncio` isn't very good.